extension/app/ts/background/simulationModeHanders.ts

```diff
@@ -55,20 +55,27 @@ export async function sendTransaction(getActiveAddressForDomain: (websiteAccess:
 			from: from,
 			chainId: await chainId,
 			nonce: await transactionCount,
-			maxFeePerGas: request.params[0].maxFeePerGas ? request.params[0].maxFeePerGas : maxFeePerGas,
-			maxPriorityFeePerGas: request.params[0].maxPriorityFeePerGas ? request.params[0].maxPriorityFeePerGas : 1n,
-			gas: request.params[0].gas ? request.params[0].gas : 90000n,
-			to: request.params[0].to ? request.params[0].to : 0n,
-			value: request.params[0].value ? request.params[0].value : 0n,
-			input: 'data' in request.params[0] && request.params[0].data !== undefined ? request.params[0].data : new Uint8Array(),
+			maxFeePerGas: sendTransactionParams.params[0].maxFeePerGas ? sendTransactionParams.params[0].maxFeePerGas : maxFeePerGas,
+			maxPriorityFeePerGas: sendTransactionParams.params[0].maxPriorityFeePerGas ? sendTransactionParams.params[0].maxPriorityFeePerGas : 1n,
+			gas: sendTransactionParams.params[0].gas ? sendTransactionParams.params[0].gas : 90000n,
```

Comment:
Not related pr but why is default gas amount 9000?

Reply:
hmm, I don't remember. Good catch, I'll create issue about it